configs/openSUSE/polkit-rules-whitelist.toml

```diff
@@ -30,10 +30,23 @@ hash = ""
 package = "polkit"
 type = "polkit"
 note = "default rule shipped by polkit, allows uid 0 to do everything"
-bug = "bsc#1125314"
+bugs = ["bsc#1125314". "bsc#1215652"]
```

Comment:
CI complains about a "unclosed array". This line seems to be the culprit

Reply:
Ah yeah, thanks, missed the period